Apps/phpanorama/panorama_connector.py

```diff
@@ -38,6 +38,7 @@ def __init__(self):
 
         self._base_url = None
         self._key = None
+        self._version = None
```

Comment:
This is the same approach used with `self._key` here. (i.e. ensure key is active => use it in the action)